web-marketplace/package.json

```diff
@@ -35,6 +35,7 @@
     "@types/react": "^18.0.23",
     "@types/react-dom": "^18.0.7",
     "@types/react-router-dom": "^5.3.3",
+    "@walletconnect/sign-client": "2.10.4",
```

Comment:
Latest version of this package was raising an error when serving the built version of the app, so kept it at a version that did not, we might want to investigate this more, potentially as part of https://regennetwork.atlassian.net/browse/APP-101?atlOrigin=eyJpIjoiNTQ5Y2Y1YmZmYmI4NGJmNjkzMjRhNjUwZDVjYzczZTEiLCJwIjoiaiJ9